tools/om-linker/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ target_link_libraries(om-linker PRIVATE
   CIRCTOMTransforms
   CIRCTSV
   CIRCTSupport
+  CIRCTVerif
```

Comment:
This makes sense, the om-linker itself is what needs the Verif dialect.  Thanks!